plugins/ingest-attachment/licenses/guava-LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Why do we need this file ? Is there a way to review and ensure it is correct or Is it meant for review from legal ?

Reply:
This is for the `dependencyLicenses` check that runs on precommit that runs validation on dependencies.  I think it was included directly from guava while imported in the original pr by @kartg.

Reply:
Thanks @mch2 for the clarification. Looks like we do maintain licensing files for dependencies, so probably it belongs to this commit. Though, I am still not sure around how to review the licensing content here. It will be good to have some guidelines around this. 

Approving, as it is backport of PR against main which is already reviewed.